services/traction/api/endpoints/models/v1/verifier.py

```diff
@@ -0,0 +1,43 @@
+"""Traction v1 API Invitations Models.
+
+
+These are the v1 API Models for managing a Tenant's Invitations.
+The underlying data is a combination of Traction specific data stored in Invitation
+table and Connection and Invitation data in AcaPy.
+
+"""
+from typing import Optional
+from uuid import UUID
+
+from pydantic import BaseModel
+
+from api.endpoints.models.v1.base import (
+    AcapyItem,
+    ListResponse,
+    GetResponse,
+)
+from api.endpoints.models.credentials import ProofRequest
+
+
+class CreatePresentationRequestPayload(BaseModel):
+    contact_id: UUID
```

Comment:
For the issuer credentials, the connection_id was also included at your and ian's suggestion so the caller could specify by contact_id or connection_id (and credential_template_id, cred_def_id). so let's stay consistent with that or we remove the other non-traction ids elsewhere.